setup/git/000_EnvSetup.rb

```diff
@@ -85,60 +59,69 @@
 action=nocheck
 # Install to the default base directory.
 basedir=default
-END
-    on host, 'pkgadd -a /root/shutupsolaris -d http://get.opencsw.org/now all'
-    on host, '/opt/csw/bin/pkgutil -U all'
-    on host, '/opt/csw/bin/pkgutil -i -y ruby19_dev'
-    on host, '/opt/csw/bin/pkgutil -i -y ruby19'
-    on host, 'ln -sf /opt/csw/bin/gem19 /usr/bin/gem'
-    on host, 'ln -sf /opt/csw/bin/ruby19 /usr/bin/ruby'
+      END
+      on host, 'pkgadd -d http://get.opencsw.org/now -a /root/shutupsolaris -n all'
+      on host, '/opt/csw/bin/pkgutil -U all'
+      on host, '/opt/csw/bin/pkgutil -y -i pkgutil'
+      on host, '/opt/csw/bin/pkgutil -y -i coreutils'
+      on host, '/opt/csw/bin/pkgutil -y -i git'
+      on host, '/opt/csw/bin/pkgutil -y -i gtar'
+    end
+    host.add_env_var('PATH', '/opt/csw/bin')
   end
 end
 
 install_packages_on(hosts, PACKAGES, :check_if_exists => true)
 
-hosts.each do |host|
-  case host['platform']
-  when /windows/
-    arch = host[:ruby_arch] || 'x86'
-    step "#{host} Selected architecture #{arch}"
+step "Install puppet-runtime" do
```

Comment:
this whole section is gross and probably should get cleaned up

Reply:
I'd love to use artifactory to fetch the runtime package, but I think we should save that for the next iteration

Reply:
Ticket opened to follow up on this, so it's not going to happen as a part of this PR